submissions/korotushak96/js-oop/js/app.js

```diff
@@ -0,0 +1,94 @@
+// Enemies our player must avoid
+const enemyWidth = 101;
+const fieldWidth = 504;
+const fieldHeight = 385;
+const finishY = 0;
+const stepY = 85;
+const stepX = 101;
+const startX=202;
+const startY=385;
+
+var Enemy = function(x,y) {
+    this.speed = Math.floor(Math.random() * (350 - 150 + 1)) + 150;
+    this.x = x;
+    this.y = y;
+    this.sprite = 'images/enemy-bug.png';
+};
+
+Enemy.prototype.isColision = function(){
+    return (player.x > this.x - enemyWidth / 2  && player.x < this.x + enemyWidth/2 &&  this.y === player.y)
+}
+
+// Update the enemy's position, required method for game
+// Parameter: dt, a time delta between ticks
+Enemy.prototype.update = function(dt) {
+    this.x = this.x < fieldWidth ? this.x + this.speed * dt : -10;
+
+    if (this.isColision()){
+        player.loose();
+    }
+};
+
+// Draw the enemy on the screen, required method for game
+Enemy.prototype.render = function() {
+    ctx.drawImage(Resources.get(this.sprite), this.x, this.y);
+};
+
+var Player = function(x,y) {
+    this.x = x;
+    this.y = y;
+    this.sprite = 'images/char-boy.png';
+};
+
+// Update the player's position, required method for game
+// Parameter: dt, a time delta between ticks
+Player.prototype.update = function() {
+    if (this.y < finishY) {
+        this.x = startX;
+        this.y=startY;
```

Comment:
Please, use pretty code. In this case you missing space around `=`. Also check yourself in all code.